.github/workflows/build_bundle_package.yml

```diff
@@ -54,16 +54,19 @@ jobs:
           cd $GITHUB_WORKSPACE/
           bash dev/ci-velox-buildstatic-centos-7.sh
       - name: Upload native libs
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
```

Comment:
It seems V3 should work on Centos7
https://github.com/apache/incubator-gluten/blob/main/.github/workflows/velox_backend.yml#L85